### ISSUE TITLE
Ember.Select stops updating value property after a couple of option changes, introduced in 1.13.4

### ISSUE THREAD

**Issue by rastapasta** [emberjs/ember.js#11817]
While being aware that Ember.Select will be deprecated soon, an odd bug is to be seen with the latest 1.13.4 release:

http://emberjs.jsbin.com/xozaco/edit?html,js,output

Given a simple select box with two options and a bound display of its current value, everything works fine until the select's option got changed 3 times - from that moment on, the bound attribute isn't updated anymore.

Kinda breaking thingies ;-)
Thanks for the amazing work on this project though!!

**Comment by raytiley** [comment]
This seems pretty bad. I kinda didn't believe the description, but its pretty easy to reproduce.

**Comment by martndemus** [comment]
Findings:
- on first load the select gets rendered using `render`
- when you change the first time the select gets rendered using `render`
- when you change the second time the select gets rendered using `rerender`
- when you change the third time no rendering is done.

**Comment by rastapasta** [comment]
Findings:

* the view's own value property continues to get set correnctly, its just the binding that seems to stop working

Extended the jsbin with two observers on the selected value - one in the extended select view, one on the bound attribute in the controller.

*[-> init]*
view observed: first
*[-> switch to second]*
controller observed: second
view observed: second
view observed: second
*[-> switch to first]*
controller observed: first
view observed: first
*[-> switch to second]*
controller observed: second
view observed: second
*[-> switch to first]*
view observed: first
*[-> switch to second]*
view observed: second

**Comment by rwjblue** [comment]
Definitely seems bad, but Ember.Select is removed from 2.0.0.  I would love to review a PR fixing this, but as of now this isn't a bug in the framework so I'm closing this issue...

**Comment by raytiley** [comment]
@rwjblue how is it not a bug in the framework? If the goal is to get people to 2.0 by first upgrading to latest 1.13 and fixing all deprecations, it seems like the built in `Ember.Select` needs to work until people get around from removing `Ember.Select`

**Comment by rwjblue** [comment]
@raytiley - It isn't a bug in the framework as of 2.0.0 because `Ember.Select` is not available in Ember 2.0.0 (therefore we cannot have `Ember.Select` bugs when it isn't part of the framework).  

I am saying that I would love to entertain a PR that fixes this, but it isn't something that Ember needs go forward since the train is moving on.

*tldr:* I'm unlikely to work on this.  Sorry for being confusing here...

**Comment by mixonic** [comment]
I would still call this a bug, though perhaps in https://github.com/emberjs/ember-legacy-views. We haven't really discussed where view bugs will be tracked. @rwjblue I'm going to re-open pending further conversation we can have offline.

**Comment by locks** [comment]
@mixonic any resolution on where the issue reporting should go?

**Comment by rwjblue** [comment]
Support (even with ember-legacy-views) is completely gone as if current beta branches. 

I do not think there is anything we can do at this point (if someone else wanted to fix they would have submitted a PR). Select was its own special kind of horrible, which is why it was removed (instead of being migrated to a component). There are a number of addons that provide much better select implementations that are fully compatible with 1.13.
